scripts/tasks.py

```diff
@@ -174,3 +174,20 @@ def stream_analysis(config):
         stream_cat = os.path.join(taskdir, f"{task.tag}.stream")
         os.system(f"cat {stream_files} >> {stream_cat}")
         logger.debug('Done!')
+
+def merge(config):
+    from btx.processing.merge import StreamtoMtz
+    setup = config.setup
+    task = config.merge
+    """ Merge reflections from stream file and convert to mtz. """
+    taskdir = os.path.join(setup.root_dir, 'merge')
+    os.makedirs(taskdir, exist_ok=True)
+    input_stream = os.path.join(setup.root_dir, f"index/{task.tag}.stream")
+    foms = task.foms.split(" ")
+    stream2mtz_obj = StreamtoMtz(input_stream, task.symmetry, taskdir, task.cell)
```

Comment:
personal preference: I would drop the `_obj` and maybe replace `2` by `to`.

Reply:
Thanks for the suggestion. Done!